search.js

```diff
@@ -0,0 +1,24 @@
+import elasticsearch from 'elasticsearch'
+import Config from './src/Config'
+
+const query = process.argv.slice(2).join(' ')
+const client = new elasticsearch.Client({ host: Config.indexUrl })
+
+const search = async () => {
+  const result = await client.search({
+    index: Config.resourceIndexName,
+    type: Config.indexType,
+    body: {
+      query: {
```

Comment:
This shows how we'd go about searching both full-text fields at the same time.

Reply:
@ndushay 	not sure I follow.

Reply:
Clarified that "full-text" is an ES term of art more or less meaning analyzed/tokenized fields.

Reply:
suggested "tokenized" may be a more readily understood term.